tests/webapp/api/test_performance_data_api.py

```diff
@@ -855,6 +897,23 @@ def test_perfcompare_results_with_only_one_run(
 
     option_collection_map = perf_compare_utils.get_option_collection_map()
 
+    response = dict()
```

Comment:
[question] would using like so be better ?
```suggestion
    response = {}
```

Reply:
I have changed it to your suggestion, I don't think it makes much of a difference though.